myapp/template.html

```diff
@@ -1,36 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <title>nies</title>
-        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-2.4.2.min.js"></script>
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.2.min.js"></script>
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.4.2.min.js"></script>
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.4.2.min.js"></script>
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-mathjax-2.4.2.min.js"></script>
-        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
-    </head>
-    <body>
-        <div class="container">
-            <p class="lead">
-                En México los salarios mínimos se establecieron en la constitución de 1917 en el
-                artículo 123, compuesto de dos partes; una se refiere a las relaciones laborales
-                entre patrón y trabajador, y otra a la relación de trabajo. A partir de la
-                instauración de la constitución mexicana es posible resaltar algunos cambios que
-                fueron dibujando y consolidando el concepto de salario mínimo en México, aunque con
-                la Constitución mexicana se institucionalizó el mercado laboral en México, es hasta
-                agosto de 1931 que se crea la primera ley reglamentaria del artículo 123, se compone
-                por las fracciones VI y VII donde se estableció que el salario mínimo deberá cubrir
-                las necesidades normales para la vida y que no es negociable su cumplimiento,
-                (Treviño 2004).
-            </p>
-        </div>
-        <div class="container">
-        <figure class="figure">
-		{{ script|safe }} {{ div|safe }}
-    <figcaption class="figure-caption text-end"><strong>Figura 1.</strong> Ingreso mensual en México en función de los años de estudio.</figcaption>
-    </figure>
-	    </div>
-    </body>
+
+<head>
+    <meta charset="utf-8">
+    <title>nies</title>
+
+    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-2.4.2.min.js"></script>
+    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.2.min.js"></script>
+    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.4.2.min.js"></script>
+    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.4.2.min.js"></script>
+    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-mathjax-2.4.2.min.js"></script>
```

Comment:
Este es otro que también ocupamos. La verdad no estoy seguro de cuál de los dos:
```suggestion
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-mathjax-2.4.2.min.js"></script>
```